<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>90</y>
      <width>221</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="actionCut"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <widget class="QMenu" name="menuExtra">
    <property name="title">
     <string>Features</string>
    </property>
    <addaction name="actionBold_Unbold"/>
    <addaction name="separator"/>
    <addaction name="actionSuperscript"/>
    <addaction name="actionSubscript"/>
    <addaction name="separator"/>
    <addaction name="actionText_Size"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About Me</string>
    </property>
    <addaction name="actionKnow_me"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuExtra"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="icon">
    <iconset>
     <normaloff>new.png</normaloff>new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset>
     <normaloff>open.png</normaloff>open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset>
     <normaloff>save.png</normaloff>save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset>
     <normaloff>saveas.png</normaloff>saveas.png</iconset>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset>
     <normaloff>cut.png</normaloff>cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset>
     <normaloff>copy.png</normaloff>copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset>
     <normaloff>paste.png</normaloff>paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset>
     <normaloff>undo.png</normaloff>undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset>
     <normaloff>redo.png</normaloff>redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionBold_Unbold">
   <property name="icon">
    <iconset>
     <normaloff>bold.png</normaloff>bold.png</iconset>
   </property>
   <property name="text">
    <string>Bold/Unbold</string>
   </property>
  </action>
  <action name="actionKnow_me">
   <property name="icon">
    <iconset>
     <normaloff>about.png</normaloff>about.png</iconset>
   </property>
   <property name="text">
    <string>Know me</string>
   </property>
  </action>
  <action name="actionSuperscript">
   <property name="icon">
    <iconset>
     <normaloff>superscript.png</normaloff>superscript.png</iconset>
   </property>
   <property name="text">
    <string>Superscript</string>
   </property>
  </action>
  <action name="actionSubscript">
   <property name="icon">
    <iconset>
     <normaloff>subscript.png</normaloff>subscript.png</iconset>
   </property>
   <property name="text">
    <string>Subscript</string>
   </property>
  </action>
  <action name="actionText_Size">
   <property name="text">
    <string>Text Size</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
